scripts/BaseL0FraxDVNScript.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: ISC
+pragma solidity ^0.8.19;
+
+import { Script } from "forge-std/Script.sol";
+import { DVN } from "@layerzerolabs/lz-evm-messagelib-v2/contracts/uln/dvn/DVN.sol";
+import { console } from "frax-std/FraxTest.sol";
+import { OptionsBuilder } from "@fraxfinance/layerzero-v2-upgradeable/oapp/contracts/oapp/libs/OptionsBuilder.sol";
+import "forge-std/StdJson.sol";
+import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
+
+abstract contract BaseL0FraxDVNScript is Script {
+    using OptionsBuilder for bytes;
+    using stdJson for string;
+    using Strings for uint256;
+
+    uint256 public dvnDeployerPK = vm.envUint("PK_DVN_DEPLOYER");
+
+    /// @dev: required to be alphabetical to conform to https://book.getfoundry.sh/cheatcodes/parse-json
+    struct L0DVNConfig {
+        string RPC;
+        address[] admins;
+        uint256 chainid;
+        uint32 eid;
+        address[] messageLibs;
+        address priceFeed;
+        uint64 quorum;
+        address[] signers;
+        uint32 vid;
```

Comment:
Is `vid` verifiable on-chain?  How do we know it is properly set when deploying?

Reply:
DVN contracts will be deployed by LZ team